### ISSUE TITLE
ERROR in saving huobi market data into redis with lastest code 

### ISSUE THREAD

**Issue by xiandong79** [bmoscon/cryptostore#104]
with lastest cryptocstore code 

config.yaml
```
    HUOBI:
        retries: 30
        l2_book:
            symbols: [BTC-USDT]
            max_depth: 300
            book_delta: true
            book_interval: 600
        trades: [BTC-USDT]
```

ERROR msg:
```
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/feedhandler.py", line 212, in _connect
    await self._handler(websocket, feed.message_handler, feed.uuid)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/feedhandler.py", line 241, in _handler
    await handler(message, self.last_msg[feed_id])
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/exchange/huobi.py", line 87, in message_handler
    await self._trade(msg, timestamp)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/exchange/huobi.py", line 73, in _trade
    receipt_timestamp=timestamp)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/feed.py", line 129, in callback
    await cb(**kwargs)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/backends/backend.py", line 33, in __call__
    await self.write(feed, pair, timestamp, receipt_timestamp, data)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/backends/redis.py", line 39, in write
    await self.redis.xadd(f"{self.key}-{feed}-{pair}", data)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/commands/streams.py", line 105, in xadd
    return self.execute(b'XADD', stream, *args)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/commands/__init__.py", line 51, in execute
    return self._pool_or_conn.execute(command, *args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/pool.py", line 196, in execute
    fut = conn.execute(command, *args, **kw)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/connection.py", line 351, in execute
    self._writer.write(encode_command(command, *args))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/util.py", line 39, in encode_command
    " float, int, or str type".format(arg))
TypeError: Argument Decimal('11154522110086674752216772') expected to be of bytearray, bytes, float, int, or str type
2020-08-24 07:09:28,582 : ERROR : HUOBI: encountered an exception, reconnecting
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/util.py", line 35, in encode_command
    barg = _converters[type(arg)](arg)
KeyError: <class 'decimal.Decimal'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/feedhandler.py", line 212, in _connect
    await self._handler(websocket, feed.message_handler, feed.uuid)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/feedhandler.py", line 241, in _handler
            max_depth: 300
    await handler(message, self.last_msg[feed_id])
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/exchange/huobi.py", line 87, in message_handler
    await self._trade(msg, timestamp)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/exchange/huobi.py", line 73, in _trade
    receipt_timestamp=timestamp)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/feed.py", line 129, in callback
    await cb(**kwargs)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/backends/backend.py", line 33, in __call__
    await self.write(feed, pair, timestamp, receipt_timestamp, data)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/cryptofeed/backends/redis.py", line 39, in write
    await self.redis.xadd(f"{self.key}-{feed}-{pair}", data)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/commands/streams.py", line 105, in xadd
    return self.execute(b'XADD', stream, *args)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/commands/__init__.py", line 51, in execute
    return self._pool_or_conn.execute(command, *args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/pool.py", line 196, in execute
    fut = conn.execute(command, *args, **kw)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/connection.py", line 351, in execute
    self._writer.write(encode_command(command, *args))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aioredis/util.py", line 39, in encode_command
    " float, int, or str type".format(arg))
TypeError: Argument Decimal('11154522247486674592663270') expected to be of bytearray, bytes, float, int, or str type
```

**Comment by yohplala** [comment]
Hi @xiandong79 
The error you report appears to me to be the same than this I reported some days ago: #95
@bmoscon solved it by a commit in Cryptofeed.
Please, do you confirm you are using the last Cryptofeed version?

Otherwise, @bmoscon , do you think one of your last work reverted the change you made in to solve #95 
(was commit  commit [b86407a](https://github.com/bmoscon/cryptofeed/commit/b86407a04698fedbded281793b02c887ae307e13))

**Comment by xiandong79** [comment]
@yohplala 
I use Cryptofeed 1.4.1, maybe the fix is updated in Cryptofeed repo. but has not updated into pip.

**Comment by bmoscon** [comment]
it has been fixed, fix is in master, this is a dupe of #95
